posthog/hogql/database/schema/events.py

```diff
@@ -84,31 +84,31 @@ class EventsTable(Table):
         "group_0": LazyJoin(
             from_field=["$group_0"],
             join_table=GroupsTable(),
-            join_function=join_with_group_n_table(0),
+            join_function=(join_with_group_n_table, 0),
```

Comment:
Updated the closure usage pattern for join_functions so that the params are still in line but saveable from `validate_model `